socorro/cron/jobs/ftpscraper.py

```diff
@@ -313,11 +329,7 @@ def _scrape_json_releases_and_nightlies(
 
     def _insert_build(self, cursor, *args, **kwargs):
         self.config.logger.debug('adding %s', args)
-        if self.config.dry_run:
-            print "INSERT BUILD"
-            print args
-            print kwargs
-        else:
```

Comment:
They're actually not the same.

But whatever. When I go work on the buildhub script and need to compare what it's doing with ftpscraper, I'll re-add id locally. :(